builtin/credential/marathon/backend_test.go

```diff
@@ -0,0 +1,94 @@
+package marathon
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/Banno/go-marathon"
+	"github.com/hashicorp/vault/logical"
+	logicaltest "github.com/hashicorp/vault/logical/testing"
+)
+
+func TestBackend_basic(t *testing.T) {
+	logicaltest.Test(t, logicaltest.TestCase{
+		PreCheck: func() { testAccPreCheck(t) },
+		Backend:  Backend(),
+		Steps: []logicaltest.TestStep{
+			testAccStepConfig(t),
+			testAccLogin(t),
+		},
+	})
+}
+
+func testAccPreCheck(t *testing.T) {
+	if v := os.Getenv("MARATHON_HOST"); v == "" {
+		t.Fatal("MARATHON_HOST must be set for acceptance tests")
+	}
+}
+
+func testAccStepConfig(t *testing.T) logicaltest.TestStep {
+	marathonUrl := fmt.Sprintf("http://%s:8080", os.Getenv("MARATHON_HOST"))
```

Comment:
Can this just be the `MARATHON_URL`?  Then we don't have to hardcode the scheme and port in it.